actions/train/mutalisk.py

```diff
@@ -0,0 +1,16 @@
+from sc2.constants import (MUTALISK)
```

Comment:
this file needs still changes so that all properties are getting called from the correct object